R/gh_token.R

```diff
@@ -164,21 +180,162 @@ gh_auth <- function(token) {
 #' # A custom one
 #' slugify_url("https://github.acme.com")
 
-slugify_url <- function(url) {
+slugify_url <- function(url) {           # https://jane@github.uni.edu/api/v3
+  url <- get_baseurl(url)                # https://jane@github.uni.edu
```

Comment:
@gaborcsardi this is a slightly new behaviour. It means the slug is GITHUB_UNI_EDU now, instead of GITHUB_UNI_EDU_API_V3.